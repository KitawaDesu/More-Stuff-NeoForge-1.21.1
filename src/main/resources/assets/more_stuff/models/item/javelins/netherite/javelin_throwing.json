{
    "loader": "neoforge:separate_transforms",
    "base": {
        "parent": "more_stuff:item/javelins/netherite/javelin_3d_model_throwing"
    },
    "perspectives": {
        "ground": {
            "parent": "more_stuff:item/javelins/netherite/javelin_item"
        },
        "gui": {
            "parent": "more_stuff:item/javelins/netherite/javelin_item"
        },
        "fixed": {
            "parent": "more_stuff:item/javelins/netherite/javelin_item"
        },
        "head": {
            "parent": "more_stuff:item/javelins/netherite/javelin_item"
        }
    }
}