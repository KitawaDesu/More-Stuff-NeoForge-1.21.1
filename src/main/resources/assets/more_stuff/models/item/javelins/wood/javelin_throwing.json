{
    "loader": "neoforge:separate_transforms",
    "base": {
        "parent": "more_stuff:item/javelins/wood/javelin_3d_model_throwing"
    },
    "perspectives": {
        "ground": {
            "parent": "more_stuff:item/javelins/wood/javelin_item"
        },
        "gui": {
            "parent": "more_stuff:item/javelins/wood/javelin_item"
        },
        "fixed": {
            "parent": "more_stuff:item/javelins/wood/javelin_item"
        },
        "head": {
            "parent": "more_stuff:item/javelins/wood/javelin_item"
        }
    }
}