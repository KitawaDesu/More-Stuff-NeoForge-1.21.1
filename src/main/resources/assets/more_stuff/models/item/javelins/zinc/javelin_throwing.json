{
    "loader": "neoforge:separate_transforms",
    "base": {
        "parent": "more_stuff:item/javelins/zinc/javelin_3d_model_throwing"
    },
    "perspectives": {
        "ground": {
            "parent": "more_stuff:item/javelins/zinc/javelin_item"
        },
        "gui": {
            "parent": "more_stuff:item/javelins/zinc/javelin_item"
        },
        "fixed": {
            "parent": "more_stuff:item/javelins/zinc/javelin_item"
        },
        "head": {
            "parent": "more_stuff:item/javelins/zinc/javelin_item"
        }
    }
}