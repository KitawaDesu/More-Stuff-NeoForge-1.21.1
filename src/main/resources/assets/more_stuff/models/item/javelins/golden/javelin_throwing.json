{
    "loader": "neoforge:separate_transforms",
    "base": {
        "parent": "more_stuff:item/javelins/golden/javelin_3d_model_throwing"
    },
    "perspectives": {
        "ground": {
            "parent": "more_stuff:item/javelins/golden/javelin_item"
        },
        "gui": {
            "parent": "more_stuff:item/javelins/golden/javelin_item"
        },
        "fixed": {
            "parent": "more_stuff:item/javelins/golden/javelin_item"
        },
        "head": {
            "parent": "more_stuff:item/javelins/golden/javelin_item"
        }
    }
}